docs/community/maintainer/release.md

```diff
@@ -0,0 +1,179 @@
+# Trivy Release Cycle
+
+## TL;DR
+
+- Minor versions (vX.Y.0) are released every 8 weeks.
+- Normal development occurs during the first 4 weeks, followed by a 4-week code freeze phase.
```

Comment:
This might be confusing, since "normal" development can continues throughout the "code freeze" phase as well.
Also, I think the term "code freeze" might be misleading since the code is "frozen" only in the branch, so work can continue throughout the code freeze.